### DISCUSSION TITLE
How does the pregnant population work in PK-Sim?

### DISCUSSION THREAD

**Discussion by AlanL526** [Open-Systems-Pharmacology/Forum#1614] [Questions & Problems] [answered]
Dear open-systems-pharmacology community,

I am learning how to construct a pregnant PBPK model following the tutorial and the model shared by Dallmann. I have completed the construction of the spatial structure in MoBi and saved the simulation as a PKML file. After that, I constructed the pregnant population in PK-Sim. When I exported the population to CSV, I was confused that the pregnant population in the metronidazole model (tutorial project file on the official forum) included parameter values about nine additional pregnancy-specific compartments, while the pregnant population (Dallmann et al. 2017) I created in PK-Sim did not generate these values. I wanted to know how the pregnant population worked in PK-sim. Do I need to input the parameter values of these special compartments manually?
![1694422229639](https://github.com/Open-Systems-Pharmacology/Forum/assets/97157587/dc305b22-c139-4363-a26a-3c4dd524de16)

Looking forward for your help!
Many thanks!

**Comment by AlanL526** [comment]
To put it another way, when I construct a pregnant population based on a pregnant individual, the pregnancy-specific structure appears in the "User Defined Variability" section. Will the values of these structures be generated automatically based on the input weight, or will they need to be filled in by the user？
![1706838671384](https://github.com/Open-Systems-Pharmacology/Forum/assets/97157587/182fdf7c-0851-480d-91b7-b9e57aec0a86)

**Reply by AndreDlm** [reply, marked as answer]
This is unfortunately a bug. It will be fixed in version 12:
https://github.com/Open-Systems-Pharmacology/PK-Sim/issues/2865

**Reply by AlanL526** [reply]
Thank you very much for your reply! In the new version, will the csv file exported by the pregnant population have the specific value of the volume of these organs?

**Reply by AndreDlm** [reply]
Yes, if fixed, the csv file will contain the physiological values of pregnancy-specific compartments.

In the meantime, you could use the following workaround: download the portable version of an earlier PK-Sim version that does not have this bug, e.g. version [10.0](https://github.com/Open-Systems-Pharmacology/PK-Sim/releases/tag/v10.0.257) (you don't need to install the portable version, just download the folder, unzip and open `PKSim.exe`), create the pregnant population, and export it to csv. 
Then, open PK-Sim v11 (or the current version you have installed) and import the csv file as population via the context menu (`Import Population from File`)

Now, the pregnancy-specific compartments appear in the `Distribution` tab with the correct gestational-age specific values and interindividual variability:
![image](https://github.com/Open-Systems-Pharmacology/Forum/assets/26541603/0b00ade0-07a1-4bb9-93ec-bde8aef85ad4)
